indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,3139 @@
+    "continent" ~ "Continent",
+    "ECT" ~ "Ecosystem Condition Typology Class",
+    "yearAdded" ~ "Year added",
+    "yearLastUpdate" ~ "Last update",
+    .default = Variable
+   )
+  ) |>
+  filter(Variable != "authors")
+
+```
+
+<!--# The following parts are autogenerated. Do not edit. -->
+
+```{r}
+#| echo: false
+#| results: asis
+status(st)
+```
+
+::: {layout-ncol="2"}
+
+
+
+> **Recomended citation**: `r paste(auth, " ", year, ". ", name, " (ID: ", id, ") ", "v. ", version, ". ecRxiv: ", url, sep="")`
+
+> **Version**: `r version`
+
+:::
+
+```{=html}
+<details>
+<summary>Show metadata</summary>
+```
+
+```{r tbl-meta}
+#| tbl-cap: 'Indicator metadata'
+#| echo: false
+#| warning: false
+
+meta |>
+  select(Variable, Value) |>
+  kbl(col.names = NULL) 
+
+```
+
+```{=html}
+</details>
+```
+
+::: {.callout-tip collapse="true"}
+
+## Logg
+
+<!--# Update this logg with short messages for each update -->
+- 09 Apr. 2025 - Original PR
+:::
+
+
+<hr />
+
+<!--# Document you work below.  -->
+
+## 1. Summary
+Functional plant community indicators represent the response of the vascular plant species composition to environmental variables (such as available temperature, moisture, nitrogen, light, pH, soil disturbance, and others) and to environmental stressors (such as competition, abiotic stress, and disturbance) and can thus be used to describe the functional signature of plant communities by calculating community-weighted means of functional indicator values related to these environmental variables or stressors for plant communities (Diekmann 2003). The functional signature of plant communities may be indicative of ecosystem condition, depending on which functional plant indicator we look at (cf. Töpper et al. 2018). For instance, using an indicator for moisture one would find a functional signature with higher moisture values for plant communities in mires compared to e.g. grasslands or forests. Deviations in the functional signature of such an indicator beyond a certain range of indicator values (as there of course is natural variation of functional signatures within an ecosystem type) may be related to a reduction in ecological condition. For different ecosystems different environmental variables are subject to change and thus relevant for condition accounting. In principle, both too much or too little of any environmental resource can represent a driver leading to shifts in plant species composition. Thus, functional plant community indicators usually are two-sided indicators that can show deviations of a plant community towards both excess or limiting amounts of a resource. One example of an exception here is the indicator for heat-requirement in alpine systems, which is designed as a one-sided indicator for a one-directional environmental pressure phenomenon: climate warming. This indicator does not rely on calculating community weighted means but rather on calculating cumulative abundance (see Framstad et al. 2022).
```

Comment:
In the NO_ALIEN indicator it was suggested to describe the data source and why there are 4 indicators: Maybe mention the data source ("Here we use ANO data to ..... "), and mention why there are four IDs.